src/test/java/socialite/logic/commands/RemarkCommandTest.java

```diff
@@ -34,60 +30,6 @@
             getTypicalAddressBook(), new UserPrefs(), new CommandHistory());
 
 
-    @Test
-    public void execute_addRemarkUnfilteredList_success() {
-        Person firstPerson = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        Person editedPerson = new PersonBuilder(firstPerson).withRemark(REMARK_STUB).build();
-
-        RemarkCommand remarkCommand = new RemarkCommand(INDEX_FIRST_PERSON, new Remark(editedPerson.getRemark().get()));
-
-        String expectedMessage = String.format(RemarkCommand.MESSAGE_ADD_REMARK_SUCCESS, editedPerson);
-
-        Model expectedModel =
-                new ModelManager(
-                        new AddressBook(model.getAddressBook()), new UserPrefs(), new CommandHistory());
-        expectedModel.setPerson(firstPerson, editedPerson);
-
-        assertCommandSuccess(remarkCommand, model, expectedMessage, expectedModel);
-    }
```

Comment:
Not exactly sure, but was wondering if it is be the same reason identified in 69c56de?